core/src/Revolution/modAccessPolicy.php

```diff
@@ -25,6 +25,42 @@
  */
 class modAccessPolicy extends xPDOSimpleObject
 {
+    const POLICY_RESOURCE = 'Resource';
+    const POLICY_ADMINISTRATOR = 'Administrator';
+    const POLICY_LOAD_ONLY = 'Load Only';
+    const POLICY_LOAD_LIST_VIEW = 'Load, List and View';
+    const POLICY_OBJECT = 'Object';
+    const POLICY_ELEMENT = 'Element';
+    const POLICY_CONTENT_EDITOR = 'Content Editor';
+    const POLICY_MEDIA_SOURCE_ADMIN = 'Media Source Admin';
+    const POLICY_MEDIA_SOURCE_USER = 'Media Source User';
+    const POLICY_DEVELOPER = 'Developer';
+    const POLICY_CONTEXT = 'Context';
+    const POLICY_HIDDEN_NAMESPACE = 'Hidden Namespace';
+
+    /**
+     * Returns list of core Policies
+     *
+     * @return array
+     */
+    public static function getCorePolicies()
+    {
+        return [
+            self::POLICY_RESOURCE,
+            self::POLICY_ADMINISTRATOR,
+            self::POLICY_LOAD_ONLY,
+            self::POLICY_LOAD_LIST_VIEW,
+            self::POLICY_OBJECT,
+            self::POLICY_ELEMENT ,
+            self::POLICY_CONTENT_EDITOR,
+            self::POLICY_MEDIA_SOURCE_ADMIN,
+            self::POLICY_MEDIA_SOURCE_USER,
+            self::POLICY_DEVELOPER,
+            self::POLICY_CONTEXT,
+            self::POLICY_HIDDEN_NAMESPACE,
+        ];
+    }
```

Comment:
Since the core polices don't change, have you considered using a constant for the list of core policies instead of a static method?

Reply:
I prefer method because it looks better in code, where it used, but yes, it is possible to move them to constant.
And another reason, method easier to override if needed (but I don't see any example when it can be required). :)